db/db_test2.cc

```diff
@@ -3661,6 +3661,50 @@ TEST_F(DBTest2, MultiDBParallelOpenTest) {
 }
 #endif  // OS_WIN
 
+namespace {
+class DummyOldStats : public Statistics {
+ public:
+  uint64_t getTickerCount(uint32_t /*ticker_type*/) const override { return 0; }
+  void recordTick(uint32_t /* ticker_type*/, uint64_t /* count */) override {
+    num_rt++;
+  }
+  // Set is not supported for fb303 stats, we just forward to the stats object
```

Comment:
Are we going to test the forwarding logic here?

Reply:
Thanks I will remove the comments.